utils/pod/pod.go

```diff
@@ -778,6 +778,21 @@ func GetLabel(key string, taskInfo *mesos.TaskInfo) string {
 	return ""
 }
 
+// GetLabelByFullSuffix checks if the whole key is the suffix of a label
```

Comment:
Should you just combine function GetLabelByFullSuffix with GetLabel? It can be confused if we keep both.

Reply:
Then, let me combine them. Btw, there are 70+ usages of GetLabel() across dce-go and dce-plugins.